traits/src/identities.rs

```diff
@@ -94,6 +105,11 @@ one_impl!(i64,   1i64);
 one_impl!(f32, 1.0f32);
 one_impl!(f64, 1.0f64);
 
+impl<T: One> One for Wrapping<T> where Wrapping<T>: Add<Output=Wrapping<T>> + Mul<Output=Wrapping<T>> {
```

Comment:
I vaguely remember that I had to put it for some reason, but now it should indeed be removed.